gemma-core/src/main/resources/default.properties

```diff
@@ -167,6 +167,7 @@ gemma.hibernate.jdbc_fetch_size=128
 gemma.hibernate.jdbc_batch_size=32
 # Default size for batch-fetching data (adjust as needed, requires more memory!)
 gemma.hibernate.default_batch_fetch_size=100
+gemma.hibernate.use_jdbc_metadata_defaults=true
```

Comment:
I don't think it's a good idea to put the option here because it suggests it can be set in `Gemma.properties`.